.adminbro/.entry.js

```diff
@@ -1 +1 @@
-AdminBro.UserComponents = {};
+AdminBro.UserComponents = {}
```

Comment:
This is changed by AdminBro on boot I think... we may solve this either using a different admin solution or using a precommit hook to run the linter... TBD

Reply:
or ignoring this file for lint

Reply:
I checked with the adminbro's guys, this file is automatically generated to keep custom components when extending dashboards, etc. We should add it to .gititgnore

Reply:
I would prefer to keep this file (if we decide to keep on going with AdminBro) in case we want custom components